scripts/reprocess.py

```diff
@@ -0,0 +1,13 @@
+from sdc.rabbit import QueuePublisher
+
+from app import settings
+
+publisher = QueuePublisher([settings.RABBIT_URLS],
+                           'sdx-survey-notification-durable')
+
+tx_id_file = 'tx_ids'
+
+with open('tx_ids', 'r') as fp:
+    lines = fp.read().splitlines()
+
+[publisher.publish_message(tx_id, headers={'tx_id': tx_id}) for tx_id in lines]
```

Comment:
I like this 👍 
- Will there ever be a need to just reprocess one?
- How do we know which one is being reprocess if something falls over? And will it stop reprocessing the rest or will it carry on and leave the one that failed there?

Reply:
I'd avoid using a list comprehension for this purpose.

As a rule of thumb a list comprehension should build a list and have side effects.